pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/RealtimeToOfflineSegmentsMinionClusterIntegrationTest.java

```diff
@@ -99,8 +103,16 @@ public void testRealtimeToOfflineSegmentsTask()
     List<SegmentZKMetadata> segmentsZKMetadata = _pinotHelixResourceManager.getSegmentsZKMetadata(_offlineTableName);
```

Comment:
`testRealtimeToOfflineSegmentsTask` was very simple. It was creating one offline segments for each task. I have just added partitioning config to table and updated the assertions.

Reply:
I think the support is not added to realtime table, so how does it work on this test?

Reply:
The `RealtimeToOfflineSegmentsTaskExecutor` uses the offline table config to create segments for offline table from realtime segments.